test/robot/replay/client.go

```diff
@@ -44,17 +45,22 @@ func (c *client) replay(ctx context.Context, t *Task) error {
 	if err := c.manager.Update(ctx, t.Action, job.Running, nil); err != nil {
 		return err
 	}
-	output, err := doReplay(ctx, t.Action, t.Input, c.store, c.tempDir)
-	status := job.Succeeded
-	if err != nil {
-		status = job.Failed
-		log.E(ctx, "Error running replay: %v", err)
-	} else if output.CallError != "" {
-		status = job.Failed
-		log.E(ctx, "Error during replay: %v", output.CallError)
-	}
+	for {
+		output, err := doReplay(ctx, t.Action, t.Input, c.store, c.tempDir)
+		status := job.Succeeded
+		if err != nil {
+			status = job.Failed
+			log.E(ctx, "Error running replay: %v", err)
+		} else if output.CallError != "" {
+			if strings.Contains(output.CallError, "text file busy") {
```

Comment:
Put this grossness in a utility function please - or declare a new `ErrTextFileBusy` error type that's returned from `shell.Target.Start()` which we can check for.

Having three magic hardcoded strings makes me three times as sad as having one.